Source/Spray/SpraySetup.cpp

```diff
@@ -73,6 +73,14 @@ getInpVal(
 void
 SprayParticleContainer::readSprayParams(int& particle_verbose)
 {
+  amrex::Print() << "\n Reading spray model parameters ..." << std::endl;
+#if AMREX_SPACEDIM == 1
+  amrex::Abort("Spray model not valid in 1D");
+#elif AMREX_SPACEDIM == 2
+  amrex::Warning(
+    " Warning: Spray model in 2D assumes narrow domain in z-direction (Lz = "
+    "dx), and is non-physical in flows with low Re");
```

Comment:
I would omit "is non-physical in flows with low Re". The results are physical for the configuration implied by the BCs, it's just that that may not be the configuration the user intends to simulate.